.travis.yml

```diff
@@ -1,4 +1,8 @@
 dist: trusty
+sudo: required
```

Comment:
It would be better to merge master to your branch. Copypasting may lead to merge conflicts